rskj-core/src/main/resources/config/rsk-sample.conf

```diff
@@ -188,18 +188,23 @@ sync {
 }
 
 rpc {
-    enabled = true
-    # Interface to bind rpc
-    # Make sure you are using the correct bind. Default value: localhost
-    address = localhost
-    port = 4444
-    # host = ["localhost", "private.ip", "external.ip", "example.com", "www.example.com"]
-
-    # A value greater than zero sets the socket value in milliseconds. Node attempts to gently close all TCP/IP connections with proper half close semantics,
-    # so a linger timeout should not be required and thus the default is -1.
-    # linger.time = 0
-
-    cors = "*.rsk.co"
+    providers : {
+        web: {
+            cors: "*.rsk.co",
```

Comment:
Please don't use this domain as default, there isn't any good reason to do this. An alternative value could be: localhost or empty.

Reply:
Now that we have a simpler configuration (#523 #530), I think we should remove the sample file.

Having said that, I think it's a good idea to update the `reference.conf` file in this PR with your suggestion.